src/migrations.zig

```diff
@@ -21,26 +21,33 @@ pub fn registerBlockMigrations(migrations: *std.StringHashMap(ZonElement)) void
 fn register(
 	collection: *std.StringHashMap([]const u8),
 	assetType: []const u8,
-	name: []const u8,
+	addonName: []const u8,
 	migrationZon: ZonElement,
 ) void {
+	const localAllocator = main.stackAllocator;
```

Comment:
I don't like aliases like this. It's not that much harder to type `main.stackAllocator`, but you don't need to jump around when reading code to figure out what `localAllocator` is doing.

Reply:
Well, then you won't like my world edit PR, cause I used it there twice, rationale there was that I was not sure if stack allocator is a good idea if in theory blueprint block array can occupy up to 1<<50 bytes (u16 x u16 x u16, 4 bytes per block). I am not sure what kind of stack allocator you have here, if it has any fallback logic when its full or is it just fixed size buffer with top pointer (?)

Reply:
The stack allocator falls back to the global allocator, and that's the general philosophy behind the `NeverFailingAllocator`.